excercises/2-locators/locators-element-all.spec.ts

```diff
@@ -22,16 +21,29 @@ describe('Protractor workshop app', function () {
 	* to see how map function can be used to verify content of multiple elements
 	*/
 
-	xit('should have Feature A, Feature B, Feature C sections ...', function () {
+	it('should have Feature A, Feature B, Feature C sections ...', async function () {
 		//TODO: User map function to verify feature names
+		const expectedText = ["Feature A", "Feature B", "Feature C"];
+		let featureText = element.all(by.css('.row.feature-box .span4 h2')).map((item, index) => item.getText());
+		expect(await featureText).toEqual(expectedText);
 	});
 
 	/**
 	 * Check http://angular.github.io/protractor/#/api?view=ElementArrayFinder
 	 * to see how fileter function can be used select elements based on condition
 	 */
-	xit('should route to "Blog" pages after selecting link', function () {
+	it('should route to "Blog" pages after selecting link', async function () {
 		//TODO: User filter function to verify feature names
+		const expectedFeature = "Blog";
+		const expectedTitle = "Protractor workshop | Blog";
+		let blogItem = element.all(by.css('ul.nav li a')).filter((elem) => {
```

Comment:
if you decided to use async, lets not use .then()

Reply:
Code updated